packages/espree/espree.js

```diff
@@ -55,6 +55,76 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+/**
+ * @import { EspreeParserCtor, EspreeParserJsxCtor } from "./lib/types.js";
+ */
+
+// ----------------------------------------------------------------------------
+// Types exported from file
+// ----------------------------------------------------------------------------
+/**
+ * @typedef {acorn.ecmaVersion} EcmaVersion
+ */
```

Comment:
I'm not sure if it's very good to define `EcmaVersion` the same way as in Acorn. Acorn adds support for new ECMA versions in minor releases (for example, ES 2026 was added in Acorn v8.15.0), but Espree needs work before a new version can be supported. See for example #658.